spec/yaml/tsg_email_activity_v3.yaml

```diff
@@ -337,16 +337,16 @@ paths:
                     description: Whether or not the outbound IP is dedicated vs shared
                     $ref: '#/components/schemas/OutboundIpType'
                 required:
+                - api_key_id
```

Comment:
parameter changes not mine

Reply:
@CristhianMotoche do you know why these changes are getting transpiled?

Reply:
Hey @tbischel @bermudezmt I'm not really sure but I'm guessing something was added in the transpiler that updates this list in alphabetical order.